src/allowed_args.cpp

```diff
@@ -245,7 +245,7 @@ static void addGeneralOptions(AllowedArgs &allowedArgs, HelpMessageMode mode)
         .addArg("dbcache=<n>", requiredInt, strprintf(_("Set database cache size in megabytes (%d to %d, default: %d)"),
                                                 nMinDbCache, nMaxDbCache, nDefaultDbCache))
         .addArg("loadblock=<file>", requiredStr, _("Imports blocks from external blk000??.dat file on startup"))
-        .addArg("maxdequesize=<n>", requiredInt, _("Keep track of the size of the most recent <n> blocks to use in fee estimation (default: %u)"), DEFAULT_MAX_DEQUE_SIZE)
+        .addArg("maxblocksizehistory=<n>", requiredInt, _("Keep track of the size of the most recent <n> blocks to use in fee estimation (default: %u)"), DEFAULT_MAX_BLOCK_HISTORY)
```

Comment:
Looks better! If I would want to continue to nag here, I'd say you should make it something like 'feemaxblocksizehistory' so that it is clear that it is for *fee* estimation.

Reply:
consider it done